package.json

```diff
@@ -20,8 +20,8 @@
   "module": "src/index.ts",
   "scripts": {
     "pretty": "yarn prettier '**/*.*'",
-    "lint": "yarn build && yarn link-plugin && yarn pretty --check && yarn eslint .",
-    "format": "yarn build && yarn link-plugin && yarn pretty --write && yarn eslint --fix .",
+    "lint": "yarn build && yarn link-plugin && yarn pretty --check && yarn eslint . --ext .ts",
+    "format": "yarn build && yarn link-plugin && yarn pretty --write && yarn eslint --fix . --ext .ts",
```

Comment:
If you add this suffix to the `--fix` will it still fix `.js` files as well?

Reply:
Good catch. I think that it will not run on js (but only 67% sure on that).

I think we might also need to look at the order or `eslint` options there.

```
eslint [options] [file|dir|glob]*
```

We're using `.` as the `dir` so I think we'll want it to look like this...

```
yarn eslint --ext .ts,.js .
yarn eslint --fix --ext .ts,.js .
```

It's possible that ESLint handles this gracefully, but I think the order above is

Reply:
Agreed, updating

Reply:
Nice catch @hockeybuggy